docs/superpowers/plans/2026_09_01_constraint_module.md

```
@@ -0,0 +1,699 @@
+		//nolint:S8148 // NOSONAR: invalid validation tag values are silently skipped
+		if minVal, err := parseNumeric(value); err == nil {
+			s.setMinimum(boundFromParsed(minVal, false))
+			return true
+		}
+	}
+	return false
+}
+```
+
+Mapping of the old emissions:
+| old handler emits | new call |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)